### PR TITLE
notifications page

### LINKED ISSUE #1741: Component: Notifications Centre
AC:

- [ ] Just the list of items with checkboxes, without the floating action bar and managing checkboxes

[Link to Figma >](https://www.figma.com/file/Goequb9eZDL6WxK7T74oxS/Notifications-(Copy)?node-id=901%3A94454)

Screenshot 2022-01-20 at 12 08 15

### PR DESCRIPTION
**Pull request by rafalpawlow**
closes #1741 and #2103

Designs:
https://www.figma.com/file/Goequb9eZDL6WxK7T74oxS/Notifications?node-id=105%3A202

### CODE REVIEW

File: packages/atlas/src/components/NotificationTile/NotificationTile.styles.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: ${variant === 'default' ? 4 : 2}px;
+      height: 100%;
+      background-color: ${cVar('colorBackgroundPrimary')};
+    }
+  `
+
+export const Wrapper = styled.div<NotificationWrapperProps>`
```
Comment (drillprop) [inline packages/atlas/src/components/NotificationTile/NotificationTile.styles.ts]: Im getting: 
image
I think it comes from here

File: packages/atlas/src/components/NotificationTile/NotificationTile.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  read,
+  selected = false,
+  variant = 'default',
+  className,
+}) => {
+  const titleVariant = variant === 'default' ? 'h300' : 't200-strong'
+  const avatarSkeletonSize = variant === 'default' ? 40 : 32
+  return (
+    <Wrapper read={read} selected={selected} loading={loading} className={className} variant={variant}>
+      {variant === 'default' && <Checkbox onChange={() => onSelect?.(id)} disabled={loading} value={selected} />}
```
Comment (drillprop) [inline packages/atlas/src/components/NotificationTile/NotificationTile.tsx]: The design specifies that should be SkeletonLoader here when loading. Not disabled Checkbox. 
Yours: 
image
Design: 
image

File: packages/atlas/src/components/NotificationTile/NotificationTile.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import { NotificationProps, NotificationTile } from '.'
+
+export default {
+  title: 'Other/NotificationTile',
+  component: NotificationTile,
+  argTypes: {
+    variant: {
+      control: { type: 'radio', options: ['default', 'compact'] },
+    },
+  },
```
Comment (drillprop) [inline packages/atlas/src/components/NotificationTile/NotificationTile.stories.tsx]:
```suggestion
```
I'm pretty sure that we don't need to add this anymore. This control should be added automatically.

File: packages/atlas/src/components/NotificationTile/NotificationTile.styles.ts [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      `
+    }
+    return css`
+      background-color: ${cVar('colorBackgroundMutedAlpha')};
+
+      :hover {
+        background-color: ${cVar('colorBackgroundStrong')};
+      }
+    `
+  }
```
Comment (drillprop) [inline packages/atlas/src/components/NotificationTile/NotificationTile.styles.ts]:
```suggestion
  if (!read) {
    return css`
      background-color: ${variant === 'default' ? cVar('colorBackground') : cVar('colorBackgroundMutedAlpha')};

      :hover {
        background-color: ${cVar('colorBackgroundStrong')};
      }
    `
  }
```
Correct me if I'm wrong, but I think we could shorten this.

File: packages/atlas/src/views/notifications/Notifications.tsx [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  const { selectedNotifications, toggleSelected } = useSelectedNotifications()
+  const smMatch = useMediaMatch('sm')
+  return (
+    <StyledLayoutGrid>
+      <GridItem colSpan={{ xxs: 12, md: 10, lg: 8 }} colStart={{ md: 2, lg: 3 }}>
+        <Header>
+          <Text variant={smMatch ? 'h700' : 'h600'}>Notifications</Text>
+          {!!UNREADS && (
+            <>
+              <StyledPill label={`${UNREADS} unread`} />
```
Comment (drillprop) [inline packages/atlas/src/views/notifications/Notifications.tsx]: I know that we have only dummy data for now, but shouldn't we get a number of unread from `NOTIFICATIONS`?

File: packages/atlas/src/views/notifications/Notifications.tsx [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    date: 'Nov 13',
+    avatarUrl: 'https://placedog.net/400/400?random&1',
+    videoTitle: 'Know your Councils #02 (CheOmsk)',
+    read: false,
+  },
+  {
+    id: '1',
+    author: 'Joystream movie',
+    text: 'withdrew their bid',
+    date: 'Nov 13',
```
Comment (drillprop) [inline packages/atlas/src/views/notifications/Notifications.tsx]: This prop actually isn't working well. What if we pass the real date here?

File: packages/atlas/src/components/NotificationTile/NotificationTile.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+              {date} • {videoTitle}
+            </Text>
+          </>
+        ) : (
+          <>
+            <SkeletonLoader width="40%" height={variant === 'default' ? 24 : 20} bottomSpace={2} />
+            <SkeletonLoader width="50%" height={variant === 'default' ? 20 : 19} />
+          </>
+        )}
+      </Content>
```
Comment (drillprop) [inline packages/atlas/src/components/NotificationTile/NotificationTile.tsx]:
```suggestion
      {!loading ? (
        <Content>
          <Title>
            <Text as="span" variant={titleVariant} secondary>
              {`${author} `}
            </Text>
            <Text as="span" variant={titleVariant}>
              {text}
            </Text>
          </Title>
          <Text variant={variant === 'default' ? 't200' : 't100'} secondary>
            {date} • {videoTitle}
          </Text>
        </Content>
      ) : (
        <Content>
          <SkeletonLoader width="40%" height={variant === 'default' ? 24 : 20} bottomSpace={2} />
          <SkeletonLoader width="50%" height={variant === 'default' ? 20 : 19} />
        </Content>
      )}
```
I removed fragments here. You could disagree with me, but in my opinion, it feels more readable for me.

**Review by drillprop — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/NotificationTile/NotificationTile.tsx (hunk 7 above)
Comment (rafalpawlow) [reply packages/atlas/src/components/NotificationTile/NotificationTile.tsx]: I disagree, but that's fine too 😛

File: packages/atlas/src/components/NotificationTile/NotificationTile.tsx [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    }
+    return formatDateAgo(date)
+  }, [date])
+
+  return (
+    <Wrapper read={read} selected={selected} loading={loading} className={className} variant={variant}>
+      {!loading ? (
+        variant === 'default' && <Checkbox onChange={() => onSelect?.(id)} value={selected} />
+      ) : (
+        <SkeletonLoader width={16} height={16} />
```
Comment (drillprop) [inline packages/atlas/src/components/NotificationTile/NotificationTile.tsx]: You also need to add border-radius.
yours:
image
design: 
image
I think it should be 2px

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]

**Comment by rafalpawlow** [comment]
After discussion with designers I decided to change it to draft. It turned out that we should use `ListItem` component for widget notifications.

**Pull request merged**